### PR TITLE
Fix intercom position

### LINKED ISSUE #260: [Bug] Overlapping placeholder intercom icon with Intercom widget
### Describe the bug

The intercom chat widget and the intercom placeholder icon don't overlap perfectly when the widget loads.

### How To Reproduce

current environment

- nextjs v13.1.1
- react-live-chat-loader v2.8.1

there is no custom styling to the intercom icon

<LiveChatLoaderProvider providerKey={appConfig.IntercomChatbotID} provider="intercom">
   <Component {...pageProps} />
  <Intercom />
</LiveChatLoaderProvider>

### Screenshots

image

### Relevant Log Output

_No response_

### Code of Conduct

- [X] I agree to follow this project’s Code of Conduct

### PR DESCRIPTION
**Pull request by amarnath-cohesive**
👋🏻 Hi there! 

This template will help you describe the work you’ve done, so one of the project maintainers can review and potentially accept it promptly. Please fill relevant sections below and feel free to delete what does not apply to your work (including this comment).

Thank you for helping us improve React Live Chat Loader! ⚡️

***

## What does this PR introduce?
In a few bullet points, please describe the changes this Pull Request makes. e.g.:

- fixes #260 .
- adds a style prop to the intercom widget which will allow setting the `left` and `right` CSS props by the user.

### CODE REVIEW

**Comment by benschwarz** [comment]
Hi @amarnath-cohesive, could you please add a description of the changes you've made and why?

File: src/types.ts [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import { CSSProperties } from 'react'
```
Comment (robmorieson) [inline src/types.ts:1]: This import isn't necessary — it currently throws a warning when running `npm run build`

File: src/components/Intercom/index.tsx [hunk 2]
```diff
@@ -77,7 +79,8 @@ const Intercom = ({
     <div
       className={containerClass}
       style={{
-        ...styles.wrapper
+        ...styles.wrapper,
+        ...wrapperStyleProps
```
Comment (robmorieson) [inline src/components/Intercom/index.tsx:83]: In testing this I wasn't able to affect the position of the placeholder icon with the styles on the container.  

If I moved `...wrapperStyleProps` down to the `div` with `role="button"` below it worked.   Perhaps the styles would make more sense there?

**Review by robmorieson — CHANGES_REQUESTED** [review]
@amarnath-cohesive thanks for your contribution!  This approach of being able to customise the placeholder icon styles for Intercom is an ideal solution to #260 as the issue is inconsistent so we don't want to change the default styles for everyone. 

I've left a couple of pointers in the code review.  

It might also be helpful if this new prop is included in the docs under the [Intercom provider section](https://github.com/amarnath-cohesive/react-live-chat-loader/blob/9f52ea466ea2d5508b93336ef71b0fc3904420f8/README.md?plain=1#L199).

**Comment by benschwarz** [comment]
Hey @amarnath-cohesive, seeing as there's no response here and work still required we'll take it that you've moved on to other things. If not, let me know and we'll re-discuss in the future. Thanks!

**Pull request closed without merging**